README.md

```diff
@@ -1,13 +1,16 @@
 # Project Name
-
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Guess who project with characters in an array - learning arrays and objects.
+The assignment was to set a a secret person by method 'Math random" and then choose questions from a drop down in HTLM. Filtering out the remaining characters based on the secret persons category and value. Giving proper alerts in correct sections of HTML.
 
 ## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+First part of the week I spent to read up on arrays and objects. Then i followed the STEPS very carefully. I consoled log the secret person to be able to check the questions and alerts.
+I tried to invent the approach by myself at first, if not working I googled, checked into other peoples code and searched for SO questions by filtering Guess who tag.
+Since this is difficult to learn I planned to only do the basics / needs for the blue requirements and not to spend time on styling.
+If I had more time I would have add a counter.
```

Comment:
You wrote the ReadMe very well,  it shows that you thought it through! Great job!